src/status_im/ui/screens/screens.cljs

```diff
@@ -117,7 +117,12 @@
             [status-im.ui.screens.wallet.send.views :as wallet.send]
             [status-im.ui.screens.wallet.manage-connections.views :as manage-all-connections]
             [status-im.ui2.screens.chat.group-details.view :as group-details]
-            [status-im.ui2.screens.chat.photo-selector.view :as photo-selector]))
+            [status-im.ui2.screens.chat.photo-selector.view :as photo-selector]
+            [status-im.ui.screens.syncing.views :as settings-syncing]))
+;[quo2.foundations.colors :as quo2.colors]))
+
+(def components
```

Comment:
is this needed?

Reply:
nope, good eyes, funny how this sneaked in, will remove :)